[Quickstarts] `k8s:build` failing with maven Micronaut Customized Image Quickstart due to wrong `jkube.image.version`

### Description
<!--
Thank you for trying out Eclipse JKube!

Please fill out the following information to help us in analyzing the issue, but feel free to skip it if you don't have the information at hand or if it does not apply. Please remove every section which does not apply to the issue.
-->

I was testing my fix for #1195 when I realized I wasn't able to build image of the `quickstarts/maven/micronaut-customized-image` since it wasn't able to pull base image

```
[ERROR] k8s: Failed to execute the build [Error while trying to build the image: Unable to add tag [org.eclipse.jkube.quickstarts.maven/micronaut-customized-image:latest] to image [org.eclipse.jkube.quickstarts.maven/micronaut-customized-image] : {"cause":"image not known","message":"failed to find image org.eclipse.jkube.quickstarts.maven/micronaut-customized-image:latest: org.eclipse.jkube.quickstarts.maven/micronaut-customized-image:latest: image not known","response":404} (Not Found: 404)]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  12.663 s
[INFO] Finished at: 2022-01-24T17:39:14+05:30
[INFO] ------------------------------------------------------------------------
```

Looks like the version of the specified [image](https://quay.io/repository/jkube/jkube-java?tag=latest&tab=tags ) is outdated:
https://github.com/eclipse/jkube/blob/d8825b7a91221125f939ed5b8a88baa601afc869/quickstarts/maven/micronaut-customized-image/pom.xml#L44

Updating it to 0.0.13 should fix this issue.

I also found reference to this same property in `quickstarts/maven/micronaut` but couldn't find it's usages. I'd suggest removing it:
https://github.com/eclipse/jkube/blob/d8825b7a91221125f939ed5b8a88baa601afc869/quickstarts/maven/micronaut/pom.xml#L39

--- a/quickstarts/maven/micronaut-customized-image/pom.xml
+++ b/quickstarts/maven/micronaut-customized-image/pom.xml
@@ -41,7 +41,7 @@
     <maven.compiler.source>11</maven.compiler.source>
     <maven.compiler.target>11</maven.compiler.target>
     <jkube.version>${project.version}</jkube.version>
-    <jkube.image.version>0.0.9</jkube.image.version>
+    <jkube.image.version>0.0.13</jkube.image.version>
     <micronaut.version>${parent.version}</micronaut.version>
     <exec.mainClass>org.eclipse.jkube.quickstart.micronaut.custom.Application</exec.mainClass>
   </properties>

--- a/quickstarts/maven/micronaut/pom.xml
+++ b/quickstarts/maven/micronaut/pom.xml
@@ -36,7 +36,6 @@
     <maven.compiler.source>11</maven.compiler.source>
     <maven.compiler.target>11</maven.compiler.target>
     <jkube.version>${project.version}</jkube.version>
-    <jkube.image.version>0.0.9</jkube.image.version>
     <micronaut.version>${project.parent.version}</micronaut.version>
     <exec.mainClass>org.eclipse.jkube.quickstart.micronaut.custom.Application</exec.mainClass>
   </properties>

